src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Deadline type of task
+ */
+public class Deadlines extends Task {
+    protected String ddl;
```

Comment:
Some people consider even protected fields "exposed", so getters and setters might be more appropriate

Reply:
fixed!